usersprompt.js

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
This code shouldn't appear in this PR. You should have created this branch off master

Reply:
will remember to do that next time